src/Logger/Adapter/HoneyBadger.php

```diff
@@ -52,6 +52,25 @@ public function push(Log $log): int
             ]);
         }
 
+        $stackFrames = [];
+
+        if (isset($log->getExtra()['stackTrace'])) {
+            $detailedTrace = $log->getExtra()['stackTrace'];
+            if (!is_array($detailedTrace)) {
+                throw new Exception("stackTrace must be an array");
+            }
+            foreach ($detailedTrace as $trace) {
+                if (!is_array($trace)) {
+                    throw new Exception("stackTrace must be an array of arrays");
+                }
+                \array_push($stackFrames, [
+                    'number' => $trace['number'],
+                    'file' => $trace['file'],
+                    'method' => $trace['method'],
+                ]);
+            }
+        }
```

Comment:
Ideally, we'd want to pass the same thing into a sentry or honeybadger log, but it would show correctly in the respective provider.

Reply:
I couldn't understand what are you trying to say here

Reply:
@2002Bishwajeet, in Appwrite, if we switch from sentry to honey badger, will honey badger still get the stack trace without modifying any of the code in Appwrite?